src/components/VFooter/VFooter.vue

```diff
@@ -134,7 +138,7 @@ export default defineComponent({
 }
 
 .footer.footer-sm .language {
-  @apply w-50;
+  @apply max-w-[12.5rem];
 }
```

Comment:
Is this nested + named class necessary for specificity purposes? If so could you add a comment? Otherwise, I wonder why @apply is being used for a single class.

Reply:
This nested class is needed because it only needs to apply at the `sm` breakpoint on the `.footer` component (whose responsiveness is currently being managed by JS). It's also done so that we can be consistent with the other classes in the same style block.